pwa.php

```diff
@@ -267,7 +271,6 @@ function _pwa_activate_plugin() {
  * Plugin deactivation hook.
  */
 function _pwa_deactivate_plugin() {
-	/* @var WP_Rewrite $wp_rewrite */
```

Comment:
See above comment.

Reply:
Done: Done in 8749128916c9807474f8bd6671ff654ff6dafc3f.